### ISSUE TITLE
runtime error when compiling payloads

### ISSUE THREAD

**Issue by jfmaes** [optiv/ScareCrow#3]
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x6b7193]

goroutine 1 [running]:
ScareCrow/limelighter.GenerateCert(0x7ffee0ffe6d3, 0x8)
	/home/jean/Documents/Tools/ScareCrow/limelighter/limelighter.go:49 +0x133
ScareCrow/limelighter.Signer(0x7ffee0ffe6d3, 0x8, 0x0, 0x0, 0x0, 0x0, 0xc000309100, 0x8)
	/home/jean/Documents/Tools/ScareCrow/limelighter/limelighter.go:1015 +0x5de
main.execute(0xc000012120, 0x73f3a9, 0x4)
	/home/jean/Documents/Tools/ScareCrow/ScareCrow.go:83 +0x63b
main.main()
	/home/jean/Documents/Tools/ScareCrow/ScareCrow.go:162 +0x90c
                                                                                           
```

**Comment by Tylous** [comment]
It looks like your using a domain that doesn't exist or is not Internet-accessible but it's hard to confirm without knowing the domain name. The limelight lib pulls the attributes from SSL certs. Please see  #1 for further details.

**Comment by Tylous** [comment]
The issue has been addressed in patch 1.3. A verbose error message now exists to explain what to do to resolve this.
